sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/DigitalTwinsModelData.cs

```diff
@@ -1,23 +1,25 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
 using Azure.Core;
 
 namespace Azure.DigitalTwins.Core
 {
     [CodeGenModel("DigitalTwinsModelData")]
     public partial class DigitalTwinsModelData
     {
-        // This class declaration makes the generated class of the same name declare Model as a **string** rather than an **object**.
-        // It also changes the namespace and renames the type from "ModelData" to "DigitalTwinsModelData"
+        // This class declaration:
+        // - Changes the namespace and renames the type from "ModelData" to "DigitalTwinsModelData"
+        // - Makes the generated class of the same name declare Model as a **string** rather than an **object**.
+        // - Renames Model to DtdlModel.
         // Do not remove.
 
         /// <summary>
-        /// The model definition.
+        /// The model definition that conforms to the Digital Twins Definition Language (DTDL).
```

Comment:
Should we call out that this is DTDL v2?

Reply:
Addressed in another PR as this one is merged.